packages/optimistic-oracle/index.js

```diff
@@ -18,9 +18,19 @@ const { getWeb3 } = require("@uma/common");
  *     mode which will exit after the loop.
  * @param {Number} errorRetries The number of times the execution loop will re-try before throwing if an error occurs.
  * @param {Number} errorRetriesTimeout The amount of milliseconds to wait between re-try iterations on failed loops.
+ * @param {Object} [commonPriceFeedConfig] Common configuration to pass to all PriceFeeds constructed by proposer.
```

Comment:
`commonPriceFeedConfig` was not actually added as a constructor param so I added it, and also renamed it from the less descriptive `defaultPriceFeedConfig`. `common` better explains how its a config passed to all constructed pricefeeds